syntax = "proto3";

message LigarLampadaRequest {
    string device_id = 1;
}

message Empty {}

service LampService {
    rpc LigarLampada (LigarLampadaRequest) returns (Empty);
    rpc DesligarLampada (LigarLampadaRequest) returns (Empty);
}